src/applications/vaos/appointment-list/components/AppointmentsPageV2/index.jsx

```diff
@@ -179,11 +188,31 @@ export default function AppointmentsPageV2() {
 
   const history = useHistory();
 
+  let paragraphText =
+    'Below is your list of appointment requests that haven’t been scheduled yet.';
+  if (featureAppointmentList) {
+    paragraphText = 'These appointment requests haven’t been scheduled yet.';
+  } else if (featureStatusImprovement) {
+    paragraphText =
+      'Your appointment requests that haven’t been scheduled yet.';
+  }
   return (
     <PageLayout showBreadcrumbs showNeedHelp>
-      <h1 className="vads-u-flex--1 vads-u-margin-bottom--1p5 vaos-hide-for-print">
+      <h1
+        className={`xsmall-screen:${
+          isPending ? 'vads-u-margin-bottom--1' : 'vads-u-margin-bottom--2'
+        } vads-u-flex--1 vaos-hide-for-print small-screen:${
+          isPending ? 'vads-u-margin-bottom--2' : 'vads-u-margin-bottom--4'
+        }`}
+      >
```

Comment:
Code cleanup. Using 'classNames' makes the code a little clear IMO when the class list gets long and has conditionals.

Reply:
done